gradle/component.gradle

```diff
@@ -7,19 +7,6 @@ repositories {
 String path_prefix = file("../gradle/componentskt.gradle.kts").exists() ? ".." : "../.."
 
 
-android {
-//     workaround for https://youtrack.jetbrains.com/issue/KT-43944
```

Comment:
Nice that this workaround can finally be removed.
What is it that allowed you to do so? I tried on my project (where versions are matching with the ones here) but gradle still complains.